admin_manual/maintenance/manually-moving-data-folders.rst

```diff
@@ -41,15 +41,25 @@ run the SQL below:
 .. code-block:: sql
    
   UPDATE oc_storages SET id='local::/mnt/owncloud' 
-    WHERE id='local::/var/www/owncloud/data'
+    WHERE id='local::/var/www/owncloud/data';
 
-The other area to check is the `oc_jobs` table. The logrotate process may have
-hard-coded a non-standard (or old) value for the data path. To check it, run the
-SQL below and see if any results are returned:
+You next need to update the ``home`` column in the ``oc_accounts`` table. 
+This column contains the absolute path for user folders, e.g., ``/mnt/data/files/admin``.
+Assuming that the new home directory is: ``/mnt/data/files/super-admin``, and that the user’s id is 1, you could change it using the following SQL statement:
+
+.. code-block:: sql
+   
+  UPDATE oc_accounts SET home='/mnt/data/files/super-admin' 
```

Comment:
considering that users might have the tendency to copy-paste instructions, maybe remove the SQL block and replace with a sentence like "you need to update the value of the 'home' column in the oc_accounts table to match the new path" and let the admin find out themselves what queries they need to execute depending on their setup. (variations of LDAP settings might require variations of the SQL statement)